src/routes/TitlesRoute/TitlesRoute.js

```diff
@@ -65,7 +65,13 @@ const TitlesRoute = ({
   const { query, querySetter, queryGetter } = useKiwtSASQuery();
 
   const { currentPage } = usePrevNextPagination();
-  const { searchKey } = useSASQQIndex({ defaultQIndex });
+
+  /*
+   * Have the default searchKey (When no qIndex is present, meaning no checkboxes are checked) include identifier values
+   * This usage of useSASQQIndex is _not_ setting the qindex in the url,
+   * that is handled in the view via SASQ and the props handed to it.
+   */
+  const { searchKey } = useSASQQIndex({ defaultQIndex: `${defaultQIndex},identifiers.identifier.value` });
```

Comment:
As of this PR: https://github.com/folio-org/stripes-erm-components/pull/675

We can now make this clearer in the split-hook use case by changing `defaultQIndex` for `defaultSearchKey`.

This is not _necessary_ since in the absence of that prop it falls back to defaultQIndex, so I'm approving this PR anyway, but we could choose to clean these up in future if we wished and thought it provided more maintainable/understandable code. Out of scope though for this PR/Issue